src/pipeline/common/ShareMethod.groovy

```diff
@@ -112,3 +112,16 @@ def sendResultToMysql(boolean sendJenkinsBuildResults, boolean sendTestResults){
     }
 }
 
+def unstashFile(Map stash_dict, String target_dir){
+    String stash_name = stash_dict["stash_name"]
+    String stash_path = stash_dict["stash_path"]
```

Comment:
The key "stash_name" and "stash_path" is defined in stash .
If there is a unstashFile, there should be a stashFile.

Reply:
So can you explain more abou your suggestion here？

Reply:
No, I mean adding a method stashFile() and it return a dict with key "stash_name" and "stash_path"

Reply:
+1  I'd prefer to add them when necessary(next refactor iteration)

Reply:
I've similar functiion inside Manifest class :-P , but not good enough to put into Shared.groovy